src/views/evaluation/Evaluation.js

```diff
@@ -0,0 +1,122 @@
+const NAV = function (option = {}){
+    return `
+    <ul class="header headerT">
+        ${logoButton(option.Logo)}
+        ${constructNav(option)}
+    </ul>`
+}
+
+const logoButton = function(option = {}){
+    return `
+    <li class="header-elements">
+        <img class= "main-logo" src="${option}">
+    </li>`
+}
+
+const buttonNav = function(option={}, active="" , specialPlacement = ""){
+    return `
+    <li class="header-elements ${specialPlacement}">
+        <a class = "header-elements-link header-elements-linkT ${active} " href="${option.Link}"> ${option.Text}</a>
+    </li>
+    `
+}
+
+
+const constructNav = function(option = {}){
+    const currentPage = "Evaluationv2.html", result =[], specialPlacement = "Loginv2.html";
+    for(var i=0;i<option.Link.length;i++){
+        if(option.Link[i] === currentPage){
+            result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}, "active-page"))
+        }else{ 
+            if(option.Link[i] === specialPlacement){
+                result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}, "", "right-button"))
+            }else{
+                result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}))
+            }
+        }
+        
+    }
+    return result.join('')
+}
+
+const tableRows = function (option = {}){
+    return `
+    <tr>
+        <td></td>
+        <td>${option.Nume}</td>
+        <td>${option.Technology}</td>
+        <td>${option.Nivel}</td>
+        <td class="detail-button">Detail</td>
+        <td>
+            <img src="${option.ImageLink}" alt="More Deatails">
+        </td>
+    </tr>`
+}
+
+const insertHeader = function(option = {}){
+    const result = []
+    for(var i=0;i<option.length;i++){
+        result.push(`<th>${option[i]}</th>`)
+    }
+    return result.join('')
+}
+
+const tableHeader = function(option = {}){
+    console.log(option)
+    return `
+    <colgroup>
+        <col class="table-button">
+        <col span="4" >
+        <col class="table-button">
+    </colgroup>
+    <tr>
+        <th></th>
+        ${insertHeader(option)}
+        <th> </th>
+        <th> </th>
+    </tr>`
+}
+
+const tableConstructor = function (option = {}){
+    const result = []
+    result.push(tableHeader(option.tHeader))
+    for (var i=0;i<option.Nume.length;i++){
+         result.push(tableRows({Nume: option.Nume[i], Technology: option.Technology[i], Nivel: option.Nivel[i], ImageLink: option.ImageLink} ))
+    }
+    return result.join('')
+}
+
+const tableBody = function(option = {}){
+    return `
+    <table cellspacing="0" cellpadding= "0" class = "tableBlock" >
+        ${tableConstructor(option)}
+    </table>
+    `
+}
+
+const Footer = function(option = {}){
+    return `	
+    <footer class="footer footerT" >
+        <p>${option.Footer}</p>
+    </footer>`
+}
+const EvaluationPage = function(option ={}){
+    return `${NAV(option.hContent)}${tableBody(option.tContent)}${Footer(option.fContent)}`
```

Comment:
I can probably infer what you meant by the `h` / `t` / `f` prefixes for these options, but then again, I may be totally off. A good practice is to always provide useful naming for your variables / properties / APIs, in general, such that anyone visiting your code (even yourself, six months from now) can easily understand what's going on and what you were trying to do. For example, why not `option.headerContent`? `option.textContent` / `option.tableContent`? `option.footerContent`?